### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by InfraredAces**
- Section the Sidebar into relevant categories
- Rename of various pages to reflect new content and organization
- Remove redundant information between Shortcuts and Usage
- Separate and update RGB LED documentation 
- Add PS4/PS5 Compatibility FAQ
- Add Contributing Guide
- Add Templates
- Add tabs to Firmware development sections for Windows and Linux

### CODE REVIEW

File: docs/contribution-guide.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# How to Contribute
+
+Thank you for takingg the time to read this and contributing to the project!
```
Comment (Pelsin) [inline docs/contribution-guide.md:3]: Tiny misspelling: "takingg"

File: docs/contribution-guide.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Thank you for takingg the time to read this and contributing to the project! 
+
+Third-party contributions help us grow and improve GP2040-CE. We want to make the pull request and contribution process useful and easy for both contributors and maintainers. 
+
+To this end we’ve put together some guidelines for contributors to help your pull request be accepted without major changes.
+
+## Project Overview
+
+GP2040-CE is largely written in C/C++, with the a significant portion of the web configurator written in Javascript within a React framework.
```
Comment (Pelsin) [inline docs/contribution-guide.md:11]: "with the a" -> "with a"

File: docs/hotkeys.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+|       Up + Down       |   Up and Down  |
+|       Down + Up       |   Up and Down  |
+
+## Invert X Axis
+
+This hotkey will invert the X-axis of your controller (i.e. pressing the Right button will result a Left input and vice versa).
+
+## Invert Y Axis
+
+This hotkey will invert the Y-axis of your controller (i.e. pressing the Up button will result a Down input and vice versa).
```
Comment (Pelsin) [inline docs/hotkeys.md:100]: "pressing the Up button will result a Down input"
I think a preposition is missing here, "will result in a"

File: docs/faq-ps4-ps5-compatibility.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+For Playstation 5 compatibility, you will need to disable the PS4 Mode add-on and enable the PS Passthrough add-on. You will also need to ensure that under the `Settings` section you have chosen `PS4` as your input mode and picked if you want the GP2040-CE unit to function as a `controller` or as a `fightstick`. 
+
+## What is PS4 Mode?
+
+PS4 mode is an add-on that grants GP2040-CE compatibility with the Playstation 4 console. It was built using the efforts of another open source firmware project named [Passing Link](https://github.com/passinglink/passinglink). 
+
+The team behind Passing Link are in no way affiliated with the GP2040-CE project, however the work and information they've made available was greatly appreciated so we could bring this highly-requested feature to our users.
+
+For more information, refer to [PS4 Mode](web-configurator-add-ons.md#ps4-mode) in the `Web Configurator - Add-ons` section of the documention.
```
Comment (Pelsin) [inline docs/faq-ps4-ps5-compatibility.md:15]: Unsure if "documention" is correct, maybe it is 🤔

File: docs/getting-help-support.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+## Discord
+
+If you need help with something, the best place to get quick support is our [Discord Server](https://discord.gg/k2pxhke7q8). Once you've read the [pinned post](https://discord.com/channels/1049366310389289001/1132450634243772519), create a post in the [support channel](https://discord.com/channels/1049366310389289001/1132447732662018089) with the following information.
+
+1. What is the name of the device are you using?
+2. Where did you buy it? Provide a link to sales page.
+3. What version of the firmware are you currently running? Check this via the Web Configurator (e.g. 0.4.3, 0.5.0, 0.7.4, etc.)
+4. Which distribution of the firmware are you currently running? Provide the exact file name (e.g. GP2040-CE_X.X.X_Pico.uf2).
+5. What is the problem that you are having?
+6. What trouble shooting steps have you already done?
```
Comment (Pelsin) [inline docs/getting-help-support.md:12]: I think "trouble shooting" is one word

File: docs/introduction.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Introduction
+
+Your controller has a control board in it that processes your inputs and translates that to commands that your computer or game console understands. GP2040-CE is firmware that facilitates this process, providing compatibility with an number of consoles and input modes. In addition to this, there are many other useful features through the Web Configurator and the system of community developed add-ons.
```
Comment (Pelsin) [inline docs/introduction.md:3]: "providing compatibility with **an** number of consoles"

**Comment by InfraredAces** [comment]
Spelling and grammar corrections made to documentation

**Review by arntsonl — APPROVED** [review]
Very nice changes!

I'll have to see if I can fill in some code conventions, but we can leave that blank for now since we don't really have specifics. I might just put a blurb in about not changing major conventions on first commit, talking to the group, and general "play nice with others".

**Pull request merged**